basics-assignment-4/src/app/my-inner/my-inner.component.ts

```diff
@@ -0,0 +1,31 @@
+import {Component, EventEmitter, Input, Output} from '@angular/core';
+
+@Component({
+  selector: 'app-my-inner',
+  templateUrl: './my-inner.component.html',
+  styleUrls: ['./my-inner.component.css']
+})
+export class MyInnerComponent {
+
+  @Output() counterEmitter: EventEmitter<number> = new EventEmitter()
+
+  innerCounter: number = 5;
```

Comment:
Nope. I've corrected this in the group chat. this should be passed as an input from the outer component.